nemo/collections/llm/api.py

```diff
@@ -52,11 +54,23 @@
 from nemo.lightning.pytorch.callbacks import PEFT, JitTransform, ModelTransform
 from nemo.utils import logging
 from nemo.utils.get_rank import is_global_rank_zero
+from nemo.utils.import_utils import safe_import
+from nemo.utils.inprocess_restart import (
```

Comment:
Do any of these need to be import guarded?

Reply:
beyond what's covered by safe import? i assert that if nvrx must be available here if inprocess is used